Cue/app/tabs/library/LibraryListView.js

```diff
@@ -106,16 +106,32 @@ class LibraryListView extends React.Component {
   }
 
   render() {
+    let refreshControl = (
+      <RefreshControl
+        colors={[CueColors.primaryTint]}
+        refreshing={this.props.refreshing}
+        onRefresh={this.props.onSwipeToRefresh}
+      />
+    )
+
     if (this.state.dataSource.getRowCount() == 0) {
       return (
-        <EmptyView
-          icon={CueIcons.emptyLibrary}
-          titleText={'You don&rsquo;t have any decks yet.'}
-          subtitleText={Platform.OS === 'android'
-            ? 'Create one or add one to your Library from Discover in the menu.'
-            : 'Create one or add one to your Library from the Discover tab.'} />
+        <View style={{flex: 1}}>
+          <EmptyView
+            icon={CueIcons.emptyLibrary}
+            titleText={'You don&rsquo;t have any decks yet.'}
+            subtitleText={Platform.OS === 'android'
+              ? 'Create one or add one to your Library from Discover in the menu.'
+              : 'Create one or add one to your Library from the Discover tab.'} />
+          <ListView
+            style={{...StyleSheet.absoluteFillObject}}
```

Comment:
Actually you should be able to use `StyleSheet.absoluteFill` directly as the style